recipe/meta.yaml

```diff
@@ -42,7 +42,11 @@ requirements:
     - make
   host:
     - vc  # [win]
-    - sqlite  # [linux]
+    - sqlite 3.46.0 # [linux]
+
+  run:
+    - vc  # [win]
+    - sqlite >3.40,<3.50 # [linux]
```

Comment:
Doesn't this restrict the dependency on sqlite?

Reply:
I don't know how to make it work. Hopefully, for running, this will allow sqlite 3.4*